app/assets/stylesheets/modules/results-documents.scss

```diff
@@ -46,8 +46,12 @@
   }
 }
 
+.alert.eds-restricted {
+  border: 1px solid var(--stanford-cardinal);
+}
+
 .eds-restricted {
-  border-left: 4px solid var(--stanford-cardinal)!important;
+  border-left: 4px solid var(--stanford-cardinal) !important;
```

Comment:
Do we need `!important` here?

Reply:
Yes. It is needed to override the border in .alert.eds-restricted  and also the border coming from blacklight.

Reply:
Can we just put `4px solid var(--stanford-cardinal)` in the `.alert.eds-restricted`?

Reply:
Even if we did, it would break the search view, so the important has to stay.

Reply:
<img width="1020" alt="Screenshot 2025-05-05 at 3 24 00 PM" src="https://github.com/user-attachments/assets/a046f41a-b0dc-4570-bc8b-f76a0df4c8e0" />
<img width="952" alt="Screenshot 2025-05-05 at 3 23 44 PM" src="https://github.com/user-attachments/assets/891a43b1-9769-44f1-80af-c2e5649d4e79" />

Reply:
Would this be the case even if there was a `.document.eds-restricted` selector?  It seems like having these as two separate styles makes sense, because one is an alert and the other is a document.

Reply:
I am not quite sure why we would want to seperate the same style into two different selectors. If we did .document.eds-restricted I would assume we would do a comma which means we would still have to do important. i.e. 
.document.eds-restricted, .alert.eds-restricted {
border-left: ....
}

I am not quite sure what you are suggesting? The only thing this pr does is add a space. The important is already there. Shouldn't this be a separate issue?

Reply:
> "seperate the same style into two different selectors"

They aren't really the same component. They just happen to look similar right now.  I can see a future where we make the alert look like something in the component library but keep the `.document` looking like it does in this PR.   In that case it's more difficult to understand that we have this one rule that affects two very different components.

Reply:
But isn't that css? The thing they share is the same style. I don't have a problem being more specific, i.e. .document.eds-restricted, .alert.eds-restricted {} but that doesn't get rid of the important. Or do we not care anymore?

Reply:
Why wouldn't .document.eds-restricted get rid of the important?  Isn't that a higher precedence selector?

Reply:
It does not, #documents .document.eds-restricted however does. Fixed.